spec/features/users/user_profile_spec.rb

```diff
@@ -1,6 +1,29 @@
 require 'rails_helper'
 
 feature 'User profile' do
+  context 'account status badges' do
+    before do
+      sign_in_live_with_2fa(profile.user)
+    end
+
+    context 'LOA1 account' do
+      let(:profile) { create(:profile) }
+
+      it 'shows a "Basic Account" badge with a tooltip' do
+        expect(page).to have_content(t('headings.profile.basic_account'))
```

Comment:
Feature spec works! I see what you are saying, sometimes a feature spec is required to get the job done. It looks like on the PR there are still only view specs. Did you forget to push up a commit?